helm/linkerd2-app/Chart.yaml

```diff
@@ -1,8 +1,11 @@
-apiVersion: "v1"
+annotations:
```

Comment:
this file looks like it has not been restored after a failed `app-build-suite` run and needs to be restored

Reply:
Missed that. Can you do that?

Reply:
yes